src/app/shared/case-history/case-history.resolver.ts

```diff
@@ -0,0 +1,45 @@
+import { ActivatedRouteSnapshot, Resolve, Router, ParamMap } from '@angular/router';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { CasesService } from '../../core/cases/cases.service';
+import { Response } from '@angular/http';
+import 'rxjs/add/operator/catch';
+import { CaseHistoryView } from '../../core/cases/case-history-view.model';
+
+@Injectable()
+export class CaseHistoryResolver implements Resolve<CaseHistoryView> {
+
+  public static readonly PARAM_JURISDICTION_ID = 'jid';
```

Comment:
private?

Reply:
requires access in the test file, hence public.